unlock_indicator.h

```diff
@@ -14,7 +14,8 @@ typedef enum {
     STATE_PAM_IDLE = 0,   /* no PAM interaction at the moment */
     STATE_PAM_VERIFY = 1, /* currently verifying the password via PAM */
     STATE_PAM_LOCK = 2,   /* currently locking the screen */
-    STATE_PAM_WRONG = 3   /* the password was wrong */
+    STATE_PAM_WRONG = 3,  /* the password was wrong */
+    STATE_PAM_FAIL = 4    /* i3lock failed to start */
```

Comment:
Please rename STATE_PAM_FAIL to STATE_I3LOCK_LOCK_FAILED, this has nothing to do with PAM :)